scalding-commons-macros/src/main/scala/com/twitter/scalding/commons/macros/impl/TBaseOrderedSerializationImpl.scala

```diff
@@ -41,18 +41,18 @@ object TBaseOrderedBufferableImpl {
   def getMinField(clazz: Class[_]): Short =
     getTFields(clazz).map(_.id).min
 
-  def apply[T <: TBase[_, _]](c: Context)(implicit T: c.WeakTypeTag[T]): c.Expr[TBaseOrderedBufferable[T]] = {
+  def apply[T <: TBase[_, _]](c: Context)(implicit T: c.WeakTypeTag[T]): c.Expr[TBaseOrderedSerialization[T]] = {
     import c.universe._
     val tpe = T.tpe
     val res = q"""
-      val generatedMin: Short = _root_.com.twitter.scalding.commons.macros.impl.TBaseOrderedBufferableImpl.getMinField(classOf[$T])
-      new _root_.com.twitter.scalding.commons.thrift.TBaseOrderedBufferable[$T] with _root_.com.twitter.bijection.macros.MacroGenerated {
+      val generatedMin: Short = _root_.com.twitter.scalding.commons.macros.impl.TBaseOrderedSerializationImpl.getMinField(classOf[$T])
```

Comment:
might as well make this `private[this]` I guess which might slightly improve look up time below.